Dockerfile

```diff
@@ -3,11 +3,15 @@ FROM node:alpine AS base
 
 WORKDIR /app
 
-COPY . .
-
-ARG NEXT_PUBLIC_API_URI=PLUNK_API_URI
```

Comment:
This line is needed. Next.js builds the applications and hard-writes public env variables into the bundled code.

We set this as a placeholder value and replace it on startup with the actual env variable.

Reply:
Should be fixed now.